service/src/test/java/uk/gov/hmcts/reform/fpl/service/NoticeOfProceedingsServiceTest.java

```diff
@@ -31,8 +36,7 @@
 import static uk.gov.hmcts.reform.fpl.utils.CaseDataGeneratorHelper.createPopulatedChildren;
 
 @ExtendWith(SpringExtension.class)
-
-public class NoticeOfProceedingServiceTest {
+class NoticeOfProceedingsServiceTest {
     @SuppressWarnings({"membername", "AbbreviationAsWordInName"})
```

Comment:
any way we could remove these supressions?

Reply:
I've a feeling this is no longer needed...

Reply:
Confirmed, removal has no impact.